### ISSUE TITLE
Surveys: select_language value appears for specific surveys as default translation

### ISSUE THREAD

**Issue by tuxpiper** [ushahidi/platform#4741]
**Description:** "select_langauge" and "(default)" values appear for specific surveys as default translation

When there are no translation selected, the "English(default)" value should be automatically present instead in both dropdown title and list. (issue is present for older surveys)

**Attachment:**
<img width="1680" alt="Screenshot 2023-08-14 at 11 06 05" src="https://github.com/ushahidi/platform/assets/8624777/9d1c82ff-468b-4615-8041-dcd64a0aea23">

<img width="1680" alt="Screenshot 2023-08-14 at 11 07 16" src="https://github.com/ushahidi/platform/assets/8624777/69908b41-37a8-47fa-bb96-919dbc2828d7">

---
Created automatically from USH-738 (https://linear.app/ushahidi/issue/USH-738/surveys-select-language-value-appears-for-specific-surveys-as-default)

**Comment by mianakbarjan** [comment]
Hello, Can I work on this? I've come from Outreachy!

**Comment by Remi-dee** [comment]
Hello @Angamanga. I'd love to work on this. Kindly note that i am done setting up my development environment

**Comment by RWUBAKWANAYO** [comment]
Hi @Angamanga ,
As an Outreachy applicant, I've completed the setup of the project locally. I'd like to work on this issue, could you please assign it to me?



![Image](https://github.com/ushahidi/platform/assets/68381641/c01ddfc6-d51a-48ea-8158-f33d425334ae)
![Image](https://github.com/ushahidi/platform/assets/68381641/6ee3e6ef-3779-4baf-9b83-44d7f5ea616a)

**Comment by felabel** [comment]
Hi @Angamanga I am an outreachy applicant and have completed my setup. I am excited to jump on this fix, please can you assign it to me?

**Comment by Bilalshaikh15** [comment]
i am working on this assign it to me.

**Comment by Bilalshaikh15** [comment]
@tuxpiper i have done all the necessary changes but i am not able to access create post page as same as you so cant test that.

https://github.com/ushahidi/platform/assets/86535450/b32307ff-a9e6-4b36-82f0-71fe2970d1f8
